conf/simulator/jsbsim/aircraft/rotwing25.xml

```diff
@@ -0,0 +1,448 @@
+        </product>
+      </function>
+      <location unit="IN">
+        <x>21.89</x>
+        <y>0</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>0</x>
+        <y>0</y>
+        <z>-1</z>
+      </direction>
+    </force>
+
+    <force name="left_motor" frame="BODY" unit="LBS">
+      <function>
+        <product>
+          <property>fcs/left_motor_lag</property>
+          <value> 26.06 </value>
+        </product>
+      </function>
+      <location unit="IN">
+        <x>0</x>
+        <y>-25.197</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>0</x>
+        <y>0</y>
+        <z>-1</z>
+      </direction>
+    </force>
+
+    <force name="pusher_motor" frame="BODY" unit="LBS">
+      <function>
+        <product>
+          <property>fcs/pusher_lag</property>
+          <value> 25.77 </value>
+        </product>
+      </function>
+      <location unit="IN">
+        <x>0</x>
+        <y>0</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>1</x>
+        <y>0</y>
+        <z>0</z>
+      </direction>
+    </force>
+
+    <!-- Now the moment couples -->
+    <force name="front_motor_couple1" frame="BODY" unit="LBS">
+      <function>
+        <product>
+          <property>fcs/front_motor_lag</property>
+          <value> 3.19 </value>
+        </product>
+      </function>
+      <location unit="IN">
+        <x>0</x>
+        <y>19.68503935</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>1</x>
+        <y>0</y>
+        <z>0</z>
+      </direction>
+    </force>
+
+    <force name="front_motor_couple2" frame="BODY" unit="LBS">
+      <function>
+        <product>
+          <property>fcs/front_motor_lag</property>
+          <value> 3.19 </value>
+        </product>
+      </function>
+      <location unit="IN">
+        <x>0</x>
+        <y>-19.68503935</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>-1</x>
+        <y>0</y>
+        <z>0</z>
+      </direction>
+    </force>
+
+    <force name="right_motor_couple1" frame="BODY" unit="LBS">
+      <function>
+        <product>
+          <property>fcs/right_motor_lag</property>
+          <value> 3.57 </value>
+        </product>
+      </function>
+      <location unit="IN">
+        <x>0</x>
+        <y>19.68503935</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>-1</x>
+        <y>0</y>
+        <z>0</z>
+      </direction>
+    </force>
+
+    <force name="right_motor_couple2" frame="BODY" unit="LBS">
+      <function>
+        <product>
+          <property>fcs/right_motor_lag</property>
+          <value> 3.57 </value>
+        </product>
+      </function>
+      <location unit="IN">
+        <x>0</x>
+        <y>-19.68503935</y>
+        <z>0</z>
+      </location>
+      <direction>
+        <x>1</x>
+        <y>0</y>
+        <z>0</z>
+      </direction>
+    </force>
+
+    <force name="back_motor_couple1" frame="BODY" unit="LBS">
```

Comment:
@dewagter @EwoudSmeur It is now possible to use `moment` directly, it makes the configuration easier. See https://github.com/paparazzi/paparazzi/blob/master/conf/simulator/jsbsim/aircraft/simple_quad_wind.xml#L214

Reply:
That is nice, though I think it was already possible inside the aerodynamics section like this:
https://github.com/paparazzi/paparazzi/blob/master/conf/simulator/jsbsim/aircraft/Systems/aerodynamics_cyfoam.xml#L4